charts/jiralert/templates/ingress.yaml

```diff
@@ -49,10 +48,10 @@ spec:
               service:
                 name: {{ $fullName }}
                 port:
-                  number: {{ $svcPort }}
+                  number: 9097
               {{- else }}
               serviceName: {{ $fullName }}
-              servicePort: {{ $svcPort }}
+              servicePort: 9097
```

Comment:
Why hardcoded?

Reply:
Because the service is hardcoded to. Whats the reason to have to configurable?

Reply:
What if the users would like to config themselves?

Reply:
I'm not aware of any use-case for it.

Reply:
Please mention, the service https://github.com/prometheus-community/helm-charts/blob/84ab6297ac99330cf3dc3d9c68d811172582acaa/charts/jiralert/templates/service.yaml#L10-L14 is hardcoded for over one year and no one needs to just it. I guess it was from the old days, there hostport was required to expose services.